src/main/java/dataLoader/csvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataLoader;
```

Comment:
By convention, in Java, package names are all in lower case. Perhaps, we should rename the package to ```dataloaders```? as the ```Json``` loader also goes in here no?

Reply:
ok